CSS grid layout guide refresh: grid lines and template areas (#38003)
mdn/content: files/en-us/web/css/css_grid_layout/grid_layout_using_line-based_placement/index.md

## Gutters or Alleys

> CSS grid includes the ability to add gutters between column and row tracks with the {{cssxref("column-gap")}} and {{cssxref("row-gap")}} properties. These specify a gap that acts much like the {{cssxref("column-gap")}} property in multi-column layout.

Author: ```suggestion
CSS grid includes the ability to add gutters between column and row tracks with the {{cssxref("column-gap")}} and {{cssxref("row-gap")}} properties, or {{cssxref("gap")}} shorthand.
```
